pydnameth/infrastucture/load/genes.py

```diff
@@ -0,0 +1,64 @@
+from pydnameth.infrastucture.path import get_data_base_path
+import numpy as np
+import os.path
+import pickle
+from tqdm import tqdm
+from pydnameth.infrastucture.load.betas import get_line_list
+
+
+def load_genes(config):
+    fn_dict = get_data_base_path(config) + '/' + 'gene_dict.pkl'
+
+    suffix = ''
+    if bool(config.experiment.data_params):
+        suffix += '_' + str(config.experiment.get_data_params_str())
+
+    fn_txt = get_data_base_path(config) + '/' + 'betas' + suffix + '.txt'
```

Comment:
Здесь может показаться, что эти 2 имени файла относятся к одной сущности, хотя это не так.
Я бы явно указал в имени какое имя файла относится к исходным beta-значениям, а какое - к генам.
```suggestion
    fn_betas_txt = get_data_base_path(config) + '/' + 'betas' + suffix + '.txt'
```

Reply:
На это имя файла распространяется предложение выше - хранить в **cache**, а не в корне базы данных